src/main/resources/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate/config.jelly

```diff
@@ -39,6 +39,14 @@
                                   deleteCaption="Delete Environment Variable" />
     </f:entry>
 
+  <f:entry field="livenessProbe" title="${%Liveness Probe}">
```

Comment:
liveness probe should be in the advanced section, at the end
it's also showing "Liveness Probe > Exec Action should not be empty"